root/var/lib/vz/snippets/exec-cmds

```diff
@@ -27,6 +27,22 @@ exec_pci_rescan() {
   echo 1 > /sys/bus/pci/rescan
 }
 
+exec_set_haltpoll() {
+  echo "Setting haltpoll for $VMID..."
+  echo $1 > /sys/module/kvm/parameters/halt_poll_ns
+}
+
+exec_assign_interrupts() {
+  ##Wait 30s for all the interrupts to show up
```

Comment:
Can we add an `echo` that `wait` is happening?

Reply:
Yes, that'd be great